src/test/java/com/axlabs/neo/grantshares/GrantSharesGovTest.java

```diff
@@ -158,7 +155,7 @@ public void succeed_creating_and_retrieving_proposal() throws Throwable {
                 .getApplicationLog().getExecutions().get(0).getStack().get(0).getInteger().intValue();
 
         // 2. Test correct setup of the created proposal.
-        NeoInvokeFunction r = gov.callInvokeFunction(GET_PROPOSAL, asList(integer(id)));
+        NeoInvokeFunction r = gov.callInvokeFunction(GET_PROPOSAL, Collections.singletonList(integer(id)));
```

Comment:
Suggestion: Keep using asList() with single values and ignore the warnings about it for the sake of consistency and readability.